cpu/nrf52/radio/nrf802154/nrf802154.c

```diff
@@ -131,6 +131,26 @@ static void _enable_tx(void)
     NRF_RADIO->TASKS_START = 1;
  }
 
+/**
+ * @brief basic ieee802.15.4 data frame validation
+ */
+static bool _data_frame_filter(void)
+{
+    size_t psdu_len = rxbuf[0];
+    if (psdu_len < IEEE802154_FCF_LEN) {
+        return false;
+    }
+    size_t expected_len = ieee802154_get_frame_hdr_len(&rxbuf[1]);
+    if (!expected_len) {
```

Comment:
Could go into the last return.
Also, I'd prefer comparison with zero since its not a bool, but its fine to me if you like writing it this way.

Reply:
Agreed, I agree with your suggestion to compare with zero to increase readability. I'll also try to make the set of checks a bit more readable with some comments :)